src/routes/ChatFinished/stories.js

```diff
@@ -0,0 +1,26 @@
+import { action } from '@storybook/addon-actions';
+import centered from '@storybook/addon-centered';
+import { withKnobs, color, text } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react';
+import ChatFinished from './component';
+
+
+const screenCentered = (storyFn) => centered(() => (
+	<div style={{ display: 'flex', width: '365px' }}>
+		{storyFn()}
+	</div>
+));
+
+storiesOf('Screen|ChatFinished', module)
+	.addDecorator(screenCentered)
+	.addDecorator(withKnobs)
+	.add('normal', () => (
+		<ChatFinished
+			title={text('title', 'Chat Finished')}
+			color={color('color', '#C1272D')}
+			greeting={text('greeting', 'Thanks to talk with us')}
```

Comment:
This should be "Thanks for talking with us".

Reply:
Thanks @wreiske.